src/gtk/textctrl.cpp

```diff
@@ -1766,7 +1805,10 @@ void wxTextCtrl::Paste()
     wxCHECK_RET( m_text != nullptr, wxT("invalid text ctrl") );
 
     if ( IsMultiLine() )
+    {
         g_signal_emit_by_name (m_text, "paste-clipboard");
+        auto value = GetValue();
```

Comment:
`value` is assigned, but never read. This piece doesn't appear to do anything in practice.